src/common/models/Projection.ts

```diff
@@ -194,22 +194,22 @@ export class Projection {
     this.dates = dates;
 
     this.locationName = this.getLocationName(summaryWithTimeseries);
-    this.stateName = summaryWithTimeseries.stateName;
+    this.state = summaryWithTimeseries.state;
     this.isCounty = parameters.isCounty;
-    this.totalPopulation = actuals.population;
+    this.totalPopulation = summaryWithTimeseries.population;
     this.fips = summaryWithTimeseries.fips;
 
     // Set up our series data exposed via getDataset().
     this.rawDailyCases = this.deltasFromCumulatives(
       this.fillLeadingNullsWithZeros(
-        actualTimeseries.map(row => row && row.cumulativeConfirmedCases),
+        actualTimeseries.map(row => (row?.cases ? row.cases : null)),
```

Comment:
As per before, this and all the similar constructs below will break when cases is `0`.

Not sure what prompted all these changes but if we need to tweak the types to use `undefined` (or even `undefined|null`) for missing values, that would probably be fine...

Reply:
ah this makes sense.  Yeah - a few days ago I tried to tweak the types to use `undefined` and it turned into a lot bigger change set than originally intended.  My immediate reaction is just to use [isNil](https://lodash.com/docs/#isNil) to check for nil-ish values for all these cases. Does that seem alright? or happy to go down a different path also

Reply:
this is fixed with better api typing!